docs/metrics.md

```diff
@@ -26,37 +26,36 @@ HTTP code.
 
 ## Routes
 
-Routes are organized in the following fashion:
-`/_/metrics/<extension-type>/<location-name/[<metric-type>]/[<bucket>]/[<key>]?[versionId=<version-id>]`
+Routes are organized as follows:
+`/_/backbeat/api/metrics/crr/<extension-type>/<location-name/[<metric-type>]/[<bucket>]/[<key>]?[versionId=<version-id>]`
```

Comment:
'crr' is an `<extension-type>` so we don't want to add this here. Alternatively, we could remove the `<extension-type>` placeholder since 'crr' is only supported in the API (but may be extended later).

Reply:
Let's not close any doors! <extension-type> it is.

Reply:
By which I meant to say: " '<extension-type>' it is." or somesuch...